js/foundation.util.touch.js

```diff
@@ -11,25 +11,35 @@ var startPosX,
     startPosY,
     startTime,
     elapsedTime,
-    isMoving = false;
+    startEvent,
+    isMoving = false,
+    didMoved = false;
 
-function onTouchEnd() {
-  //  alert(this);
+function onTouchEnd(e) {
   this.removeEventListener('touchmove', onTouchMove);
   this.removeEventListener('touchend', onTouchEnd);
+
+  // If the touch did not moved, consider it as a "tap"
```

Comment:
Oh,

moved => move

Reply:
Thank you for the review. Did you tested the code ?